src/App.js

```diff
@@ -1,24 +1,26 @@
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import Auth from './views/Auth';
+import { useState } from 'react';
+import { getUser } from './services/users';
+import Todos from './views/Todos';
+import Header from './components/Header';
 
 function App() {
+  const [currentUser, setCurrentUser] = useState(getUser());
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Header currentUser={currentUser} setCurrentUser={setCurrentUser} />
+      <Switch>
+        <Route exact path='/'>
+          <Auth setCurrentUser={setCurrentUser} />
+        </Route>
+        <Route exact path='/todos'>
+          <Todos currentUser={currentUser} />
+        </Route>
```

Comment:
Here we will want to check for a current user and redirect to either home or todos if logged in.